Externals.cfg

```diff
@@ -33,25 +33,26 @@ repo_url = https://github.com/nmizukami/mizuRoute
 hash = 34723c2
 required = True
 
+# Note that this a branch mvertens/scol_bugfix - needs a PR to master
 [cime]
 local_path = cime
 protocol = git
 repo_url = https://github.com/ESMCI/cime
-tag = branch_tags/cime5.8.37_a02
+hash = ce458952e
```

Comment:
- [ ] We'll want to point to a tag - either a cime master tag or a branch tag

Reply:
I plan to make a branch tag. There will likely need to be more work to get a tag on main-dev.

Reply:
OK, we got a cime main-dev tag for it.